docs/application/dotnet/guides/nui/relative-layout.md

```diff
@@ -0,0 +1,82 @@
+# Relative layout
```

Comment:
Please add this guide at an appropriate location in Toc_all.md

Reply:
Added :)